src/main/java/Storage.java

```diff
@@ -50,14 +51,15 @@ protected static void addToList(Path path, String desc) {
      * @param deadline the deadline the task has to be completed by
      */
     protected static void addToList(Path path, String desc, LocalDate deadline) {
-        String line = "P" + "," + 0 + "," + desc + "," + deadline + "\n";
+        String line = "D" + "," + 0 + "," + desc + "," + deadline + "\n";
         try {
             Files.write(path, line.getBytes(), StandardOpenOption.APPEND);
         } catch (IOException e) {
             System.out.println(e.getMessage());
         }
     }
 
+
     /**
      * Writes to the file given, in the format of [type][status] [desc].
      * Eg. [T][ ] read book
```

Comment:
I like how detailed your Javadoc is, even providing the example of the format that will be written to file.